ui/js/controllers/admin/exclusions_detail.js

```diff
@@ -1,75 +0,0 @@
-'use strict';
-
-admin.controller('ExclusionsDetailCtrl', [
-    '$scope', 'ThJobExclusionModel', 'thNotify', '$state', '$stateParams',
-    'strReloadTreeherder',
```

Comment:
I think `strReloadTreeherder` in `values.js` is unused now too.

Reply:
Another nice catch.  :)